packages/build/tests/plugins/tests.js

```diff
@@ -187,6 +188,88 @@ if (!version.startsWith('v8.')) {
   })
 }
 
+test('Deploy plugin succeeds', async t => {
+  const { socketPath, requests, stopServer } = await startDeployServer()
+  try {
+    await runFixture(t, 'empty', { flags: { buildbotServerSocket: socketPath, featureFlags: 'triggerDeploy' } })
+  } finally {
+    await stopServer()
+  }
+
+  t.true(requests.every(isValidDeployReponse))
+})
+
+test('Deploy plugin is only run with ---buildbotServerSocket', async t => {
```

Comment:
I'm a bit confused by the name of this test since `triggerDeploy` is passed as a feature flag and the tests says `is only run with ---buildbotServerSocket`

Reply:
Fixed the name, please let me know whether this is clearer :)